### PR TITLE
Refresh: homepage - hero, update to black with flag (Fix#15482)

### LINKED ISSUE #15482: Refresh: Homepage hero
Please update the first component on the homepage to match [the most recent Figma](https://www.figma.com/design/umfT9uNYJwAjIwgz360BdB/MVP-3.0-Mozilla.org?node-id=949-140067&m=dev).

This component is still in flux but it's worth coding the most recent changes. There are plans to make the flag animated but work with a still image for now.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Update homepage hero to most recent Figma.

## Significant changes and points to review

- Make the homepage "hero" its own component, separate from about page
- HTML, CSS, and images to support the new "hero" on the homepage
- Add alt white and black colours

## Issue / Bugzilla link

Fix #15482

## Testing

http://localhost:8000/en-US/ with refresh switch on

### CODE REVIEW

File: media/css/m24/flag.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        img {
+            width: 100%;
+            max-width: 216px;
+        }
+    }
+}
+
+@media #{$mq-xl} {
+    .m24-c-flag {
+        @include grid;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/m24/flag.scss:64]: ✅

File: media/css/m24/flag.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+.m24-c-flag {
+    @include container;
+    padding-top: $spacer-2xl;
+    padding-bottom: $spacer-xl;
+}
+
+.m24-c-flag-title {
+    font-size: 48px;
+    line-height: 1;
+    margin: 0;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/m24/flag.scss:14]: ✅

File: media/css/m24/flag.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    text-wrap: balance;
+    text-wrap-style: balance;
+}
+
+.m24-c-flag-subtitle {
+    font-size: 18px;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/m24/flag.scss:20]: I think it's okay ✅

**Review by maureenlholland — APPROVED** [review]
💯

**Pull request merged**